packages/bpk-component-button/src/common.scss

```diff
@@ -16,6 +16,8 @@
  * limitations under the License.
```

Comment:
Shouldn't this file be called `BpkButtonBase.scss`?

Reply:
It is yeah